errno/infoschema_test.go

```diff
@@ -16,6 +16,7 @@ package errno
 import (
 	"testing"
 
+	_ "github.com/pingcap/tidb/util/testbridge"
```

Comment:
We will remove this bridge once all test cases migrated and we don't use `check.*` in test flag.

... but why can't go just ignore undefined flag...

Reply:
You can now identify if file is migrated by 'grep testbridge'.

Reply:
You can see I change the way to use `TestMain` ( https://github.com/pingcap/tidb/pull/26082/commits/9d49ce9d64d79e9f0c48f43363829711dd938314 ) so that we centralize the bridge. I think it is a more native way while developers easily miss the wrapper imports section.